lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -307,6 +307,89 @@ def test_index_no_visible_section_in_chapter(self):
         self.assertNotIn('Problem 1', response.content)
         self.assertNotIn('Problem 2', response.content)
 
+    def _create_staff_user_and_course(self):
+        """
+        User will have staff access and creates the course
+        """
+        self.user = GlobalStaffFactory.create()
+        self.client.login(username=self.user.username, password='test')
+
+        # create the course
+        course = CourseFactory.create()
```

Comment:
If you make this change, you'll also want to change the name of the method and the docstring